lib/api/core/plugins/pluginsManager.js

```diff
@@ -34,7 +34,8 @@ const
   _ = require('lodash'),
   fs = require('fs'),
   { KuzzleError } = require('kuzzle-common-objects').errors,
-  Manifest = require('./manifest');
+  Manifest = require('./manifest'),
+  loadPluginsError = require('../../../config/error-codes/index').loadPluginsError;
```

Comment:
```suggestion
  { loadPluginsError } = require('../../../config/error-codes/index');
```
Prefer destructuring assignment